main.go

```diff
@@ -72,37 +72,40 @@ func createInstallNpmCommand() (*command.Model, error) {
 		return nil, fmt.Errorf("unsupported operating system: %s", runtime.GOOS)
 	}
 
-	return command.New(args[0], args[1:]...), nil
+	return command.NewWithStandardOuts(args[0], args[1:]...), nil
 }
 
-func runAndLog(cmd *command.Model) (string, error) {
-	out, err := cmd.RunAndReturnTrimmedCombinedOutput()
+func runAndLog(cmd *command.Model) error {
```

Comment:
remove this function, you call it for 3 commands, but they should not treated in the same way.
In the case of `setNpmVersion` and `createInstallNpmCommand` we do not care about the command output, just in case of error,m while in case of the user defined npm command we care about the output anyway.

Also, you use `runAndLog` in a mixed way: for some commands, you have a creator function and you call these commands with `runAndLog` in the main function, while other command functions are solving the command creating and calling the command as well.